app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -250,7 +249,7 @@ const RentalAssistanceForm = ({ values, onSave, loading, onClose, applicationMem
   )
 }
 
-const RentalAssistance = ({ store, form }) => {
+const RentalAssistance = ({ store, form, submitting }) => {
```

Comment:
I'm not super clear on what this submitting variable is here. I was kinda expecting it to be like a loading state, but now I'm just a bit confused.

Reply:
This is being passed down from the main form, it is to let the other pieces know that the form is currently `submitting` https://final-form.org/docs/final-form/types/FormState#submitting, it is an trigger already managed by the form state instead of adding our own hook or state